src/components/Editor/Properties/PropertySelect.vue

```diff
@@ -86,3 +86,15 @@ export default {
 	},
 }
 </script>
+
+<style lang="scss" scoped>
+
+.property-select {
+	margin-bottom: 4px;
+	&__input {
+		width: calc(100% - 34px - 34px);
```

Comment:
Can you add a comment to explain the 34px come from the `__icon` & `__info` width (because there might be a case where we don't add an `icon-details` element).

Reply:
done